### PR TITLE
The Quarry touch controls

### PR DESCRIPTION
**Pull request by Kode-Z**
www xbox com_en-GB_play_(iPhone SE) (2)

Added touch controls for The Quarry.

### CODE REVIEW

**Comment by Kode-Z** [comment]
Hopefully I've uploaded it right this time! Both files have been beautified.

File: touch-layouts/1731121397.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "The Quarry",
+  "default_layout": "Custom",
```
Comment (redphx) [inline touch-layouts/1731121397.json]: `custom` (lowercase)

File: touch-layouts/layouts/the-quarry.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
+{
+  "schema_version": 1,
+  "layouts": {
+    "Custom": {
```
Comment (redphx) [inline touch-layouts/layouts/the-quarry.json]: `custom` (lowercase)

File: touch-layouts/layouts/the-quarry.json [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  "layouts": {
+    "Custom": {
+      "name": "Custom",
+      "content": {
+        "left": {
+          "inner": [
+            {
+              "type": "joystick",
+              "action": "leftTrigger",
+              "actionThreshold": 0.75,
```
Comment (redphx) [inline touch-layouts/layouts/the-quarry.json]: Have you tried setting this value to `2`? Making it a higher value so it's less likely to be triggered by accident.  

I haven't played this game yet so I don't know if it's better to have a separate sprint button.

**Review by redphx — COMMENTED** [review]
Please also add a screenshot

**Comment by Kode-Z** [comment]
> Please also add a screenshot

Done!

**Comment by redphx** [comment]
I don't think this layout can be used. It's missing some buttons: https://outsidergaming.com/the-quarry-complete-controls-guide-for-ps4-ps5-xbox-one-xbox-series-xs-and-gameplay-tips-for-beginners/

**Comment by Kode-Z** [comment]
No worries, I'll revise it.

**Comment by Kode-Z** [comment]
Just updated the layout.

File: touch-layouts/layouts/the-quarry.json (hunk 3 above)
Comment (Kode-Z) [reply touch-layouts/layouts/the-quarry.json]: I changed the value to 2. Plays much better at this setting!

**Comment by redphx** [comment]
Do the X and Y buttons do anything?

**Comment by Kode-Z** [comment]
Only for QTE events

**Comment by redphx** [comment]
I thought QTE events only use analog sticks and A button? https://www.trueachievements.com/game/The-Quarry-Xbox-One/walkthrough/2

**Comment by Kode-Z** [comment]
I'll amend the layout

**Comment by Kode-Z** [comment]
The X button is used in the menus, so I'll add it to the top row

**Comment by redphx** [comment]
- Both X and Y buttons are needed for menu
- Maybe putting LB and RB on the same side? (left)
- LT buttond does nothing
- Update the position of A & B buttons, make them feel more natural to press

**Comment by redphx** [comment]
Let me handle the rest

**Comment by Kode-Z** [comment]
image

Maybe something like this?

**Comment by redphx** [comment]
Like this. You don't need the inventory button.

image

**Comment by redphx** [comment]
Thanks!

**Comment by Kode-Z** [comment]
Thanks for finishing it. Really appreciate it.

**Pull request merged**